cot-macros/src/lib.rs

```diff
@@ -192,3 +192,82 @@ pub(crate) fn cot_ident() -> proc_macro2::TokenStream {
         }
     }
 }
+
+#[proc_macro_derive(FromRequestParts)]
+pub fn derive_from_request_parts(input: TokenStream) -> TokenStream {
```

Comment:
Please add some documentation and add a re-export in the main crate, like so: https://github.com/cot-rs/cot/blob/f9d7c43c639b9074efa9870bab1edfd96c50b29a/cot/src/lib.rs#L94-L156

Otherwise, the macro will not be easily available for the users.